Android/PowerUpApp/PowerUpApp/res/layout/activity_main.xml

```diff
@@ -0,0 +1,60 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It seems like all items are one behind the other. You should consider using a LinearLayout